quilt/tools/command.py

```diff
@@ -667,6 +666,10 @@ def install(package, hash=None, version=None, tag=None, force=False):
             if entry['hash'].startswith(hash):
                 pkghash = entry['hash']
                 break
+        else:
+            print("could not find hash {hash} for package {owner}/{pkg}.".format(
```

Comment:
This needs to be `raise CommandException(...)`, to actually indicate an error.

(Also, start messages with uppercase, just for consistency.)

Reply:
DONE.